tests/operations.go

```diff
@@ -184,3 +202,92 @@ func isNotFound(err error) bool {
 		return k8serrors.IsNotFound(err)
 	}
 }
+
+// GetHealthyCSPCount ...
+func (ops *Operations) GetHealthyCSPCount(spcName string) int {
```

Comment:
Why do we need to put these logic in operations?
Can this be inside the test code?